datadog-profiling/src/serializer/compressed_streaming_encoder.rs

```diff
@@ -3,29 +3,36 @@
 
 use super::UploadCompression;
 use bytes::BufMut;
-use lz4_flex::frame::FrameEncoder;
+use lz4_flex::frame::FrameEncoder as Lz4FrameEncoder;
 use prost::encoding::{encode_key, encode_varint, encoded_len_varint, key_len, WireType};
 use std::io::{self, Write};
+use zstd::stream::Encoder as ZstdEncoder;
 
 // None is not really for prod, so the fact it takes 0 space, creating a large
 // discrepancy in size between the enum variants, is irrelevant.
 #[allow(clippy::large_enum_variant)]
 enum Compressor {
     None,
```

Comment:
We discussed offline using `None(Vec<u8>)` to make the API the same for each type